scripts/tests/chiptest/__init__.py

```diff
@@ -160,6 +160,60 @@ def _GetInDevelopmentTests() -> Set[str]:
+    return {
+        "DL_LockUnlock",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_AddNewFabricFromExistingFabric",  # darwin-framework-tool does not support the GetCommissionerRootCertificate command.
+        # The name of the arguments once converted differs for chip-tool and darwin-framework-tool (attribute-ids vs attribute-id)
+        "TestAttributesById",
+        "TestBasicInformation",  # darwin-framework-tool does not support writing readonly attributes by name
+        "TestClusterComplexTypes",  # Darwin framework has no way to represent a present but null optional nullable field.
+        # When reading TestFabricScoped, the result differs because of missing fields that has been declared in the YAML step with null value to workaround some limitation of the test harness (#29110)
+        "TestClusterMultiFabric",
+        "TestCommandsById",  # darwin-framework-tool does not support writing readonly attributes by name
+        "TestDiagnosticLogs",  # darwin-framework-tool does not implement a BDXTransferServerDelegate
+        "TestDiscovery",  # darwin-framework-tool does not support dns-sd commands.
+        "TestEvents",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "TestEventsById",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "TestGroupMessaging",  # darwin-framework-tool does not support group commands.
+        "TestIcdManagementCluster",  # darwin-framework-tool does not support ICD registration
+        "TestUnitTestingClusterMei",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "TestReadNoneSubscribeNone",  # darwin-framework-tool does not supports those commands.
+
+        "Test_TC_ACE_1_6",  # darwin-framework-tool does not support group commands.
+        "Test_TC_ACL_2_5",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_ACL_2_6",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_ACL_2_7",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_ACL_2_8",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_ACL_2_9",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_ACL_2_10",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        "Test_TC_BINFO_2_1",  # darwin-framework-tool does not support writing readonly attributes by name
+        "Test_TC_BINFO_2_2",  # darwin-framework-tool does not currently support reading or subscribing to Events
+        # The name of the arguments once converted differs for chip-tool and darwin-framework-tool (attribute-ids vs attribute-id)
```

Comment:
Should probably have a tracking issue.

Reply:
I have opened https://github.com/project-chip/connectedhomeip/issues/31934